admin/app/router.js

```diff
@@ -1,14 +1,25 @@
 import EmberRouter from '@ember/routing/router';
+import { service } from '@ember/service';
 import config from 'pix-admin/config/environment';
 
 export default class Router extends EmberRouter {
+  @service pixToast;
+
   location = config.locationType;
   rootURL = config.rootURL;
 
   constructor() {
     super(...arguments);
     this.on('routeDidChange', (transition) => {
       if (transition.from && transition.to.name !== transition.from.name) {
+        if (
+          !transition.from.parent.name.includes(transition.to.parent.name) &&
+          !transition.from.name.includes('.edit') &&
+          !transition.from.name.includes('.new') &&
+          !transition.from.name.includes('.update')
```

Comment:
Peut-être mettre ça dans une constante pour rendre explicite ce que font toutes les conditions
```suggestion
const isTransitionInSameRootPage =  !['.edit', '.new', '.update'].every((route) => transition.from.name.includes(route));
```